internal/repotesting/repotesting.go

```diff
@@ -68,7 +68,7 @@ func (e *Environment) setup(tb testing.TB, version format.Version, opts ...Optio
 			MutableParameters: format.MutableParameters{
 				Version: version,
 			},
-			HMACSecret:           []byte{},
+			HMACSecret:           []byte("test-hmac-secret"),
```

Comment:
nit: pad to 32 bytes and update the corresponding tests 🙏🏼 
Thanks.

Reply:
👍 Updated.